themeUtils/generateThemeClass.js

```diff
@@ -0,0 +1 @@
+export default prop => `mdc-theme--${prop}-bg `;
```

Comment:
@prateekbh should we keep a space here? Shouldn't it be handled where it's used?

Reply:
makes sense